packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -29,6 +29,7 @@ export const BookingFields = ({
     // The logic here intends to make modifications to booking fields based on the way we want to specifically show Booking Form
     <div>
       {fields.map((field, index) => {
+        if (field.editable === "user-owner-secret" && currentView === "reschedule") return null;
```

Comment:
Can we add a comment here explaining why this is needed?

Reply:
@zomars Idea was that a secret question and response should not be visible to a user once booking is completed. Now since if we click on reschedule link and can see all the responses, I thought of adding this condition to not show that.